sandbox/urls.py

```diff
@@ -26,6 +26,14 @@
 
 
 urlpatterns = [
+    # Add sitemap.xml URL to the robots.txt so we don't need to register sitemap.xml from each
+    # crawler administration panel
+    path(
+        r"robots.txt",
```

Comment:
Actually this is not a regex:
```suggestion
        "robots.txt",
```